lib/styles/components/_buttons.scss

```diff
@@ -101,6 +101,11 @@
 		&__loadingIcon {
 			color: $icon;
 		}
+
+		&__loadingIcon {
```

Comment:
why do we have to move it here?

Reply:
I like to have styles for element for constant classes, not for dynamic like `.loading`. Now it's not a problem, but sometimes it can occurs problems with animations. Dynamic class styles could be used for example: in changing opacity, from default 0 to 1 in loading state.

Reply:
makes sense, but do you think `setButtonAdditions()` mixin is the right place to put it?

Reply:
I moved to the right place.